core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,17 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('updateLanguage', ['$rootScope', function($rootScope) {
```

Comment:
@seanlip I made it a directive because by using Angular translate i can just listen to the whenever the lang is changed and just update . If not we will need a function that listens to lang change and one more function that updates it , which will reduce the efficiency . 
If you still want me to change to remove the directive and update the controller, i will .